cpp/src/io/parquet/reader_impl.cu

```diff
@@ -37,6 +37,8 @@
 #include <numeric>
 #include <regex>
 
+#include <cudf_test/column_utilities.hpp>
```

Comment:
Why is a cudf test header included in a library source?

Reply:
Good catch.  Lingering debug code.